### ISSUE TITLE
Question / Help me extract data

### ISSUE THREAD

**Issue by Nex4rius** [b3dk7/StegExpose#4]
Hello,

I'm sorry this is probably not the right place.
I've run the progam and it identified the 3 stego images in the test folder but how do I extract / view the data from them?

Thanks you.

**Comment by b3dk7** [comment]
Hi,
To develop a the program i created sample stego images using four different tools. they are listed in section 3.1 in my white paper. https://arxiv.org/pdf/1410.6656v1.pdf
You can try each tool to extract data.
Please note that this study only focuses on lossless images meaning it was not designed with jpeg images in mind. 
Regards
Benedikt

**Comment by b3dk7** [comment]
if you are ever interested in creating a tool for jpeg images, let me know. i can possibly give you some tips. Unfortunately i can't develop this myself right now since i have a full time job and a kid.

**Comment by Nex4rius** [comment]
Wow that was quick :)
Thanks.

**Comment by Nex4rius** [comment]
Hi,
right now I only want to test it with the png images.

I've tried those program and I'm unable to extract the hidden data (of the testfolder).
They either tell me there is nothing hidden, it's corrupted or it requires a password.
I tried them all with an empty password. Does it only work with passwords?

Anyone help would be appreciated. :)

**Comment by b3dk7** [comment]
before embedding secrets they are often encrypted. depending on how much time you would like to invest, you could build a password cracker for each embedding tool.
heres a collection of passwords you could start off with - https://github.com/danielmiessler/SecLists/tree/master/Passwords

below im going to list a few things you should consdier
- my tool might have given you a false positive. i think its about 90% accurate on high embedding rates. this accuracy drops when the secret message has a very small file size or the image is not photographic. the study was only done on photographic images and not digitally drawn images.
- a different embedding tool might be used or the embedding tool might not be publicly available

so basically there is the risk that  you spend time on this and never extract any secrets. I would only bother if the image is photographic.

**Comment by LabunskyA** [comment]
If you want help extracting data, I suggest posting the pics to r/Steganography. And link it here, maybe someone will figure something out in their free time

**Comment by Nex4rius** [comment]
Thanks you for your help :)

**Comment by lr1734a** [comment]
Hello! I came across your code and thought that it was super cool! I decided to study it a bit for my Capstone project and am going to attempt to make a program that is similar to it. I hope to improve and optimize detection of steganographic images(if I have enough time, maybe even handling JPEG).  I was wondering if there are any other times when accuracy might drop? How were the approximate suspicious amount of bytes calculated? Did you find that different kinds of images (landscape, black/white, portraits, animals) are harder to detect? The answers to these questions and any other insight that you are willing to provide would be very helpful. I appreciate your time! Thank you!
